src/jshint.js

```diff
@@ -3413,88 +3409,31 @@ var JSHINT = (function() {
     });
   }
 
-  var conststatement = stmt("const", function(context) {
+  function variableStatement(type, statement, context) {
```

Comment:
Don't do this --- it's going to painfully bitrot some code of mine, and there are big differences between lexical declarations and var declarations. In particular, binding resolution for var statements is different from lexical declarations, so lets keep them separate.